### ISSUE TITLE
UML 15x15  strucks on point 149 with version 1.3.0

### ISSUE THREAD

**Issue by Sherman-c** [Jyers/Marlin#446]
Hi 
I encounter another strange issue with E3V2-UBL-BLTouch-15x15-v4.2.7.bin, at first, it seem ran ok, it completed with only 17x points were saved to eeprom.  Since the bed level test was unsuccessful, I cleared the mesh data, and attempted to recreate new one again, this time it struck on point 130, repeated to test point 149.  BLtouch issue?

Software 1.30 E3V2-UBL-BLTouch-15x15-v4.2.7.bin,
Bed level test starts Gcode
G28
M420 S1

Regards
Sherman

PS I'm going to test 10x10 UBL and see how it goes

Due to the size of the video file, I reduced the image quality to 1.5mb/s

https://user-images.githubusercontent.com/68487284/114295203-fd6f0880-9ae6-11eb-961a-c87e4112804e.mp4

**Comment by Sherman-c** [comment]
[](url)

**Comment by Sherman-c** [comment]
Sorry I pressed the wrong button

**Comment by tititopher68** [comment]
Hello,

No issue
;-) just look at : https://github.com/Jyers/Marlin/issues/403

**Comment by tititopher68** [comment]
Hello,

No issue
;-) just look at : https://github.com/Jyers/Marlin/issues/403

**Comment by Sherman-c** [comment]
Thanks for clearing that up for me.


Cheers
Sherman

**Comment by Jyers** [comment]
Sorry, I think @tititopher68 misunderstood. You definitely have a problem here. I looks to me like a probe failure of some kind. The most common cause of this would be a connection issue of some kind. If you check all the connectors and still face this issue, you should also try adjusting the set screw on the top of the BL Touch, as that is a crucial component and can easily be slightly too far in or out.

**Comment by tititopher68** [comment]
Sorry, yes, @Jyers is absolutely right ... I am using a translator ... bad translation.

**Comment by Sherman-c** [comment]
Hi Jake,

Thanks for the update, the problem seems gone away with UBL 10x10 without
touching anything else.  I hot glued the connector when I installed the
BLTouch.  Since I have problem with UBL, both 10x10 and 15x15 versions,  it
printed on air on certain lines near the center.  I switched to BLTouch
5x5, and it was working ok so far.  I will send you a video once I get my
raspberry pi box printed.

Cheers
Sherman
